src/gentropy/susie_finemapper.py

```diff
@@ -142,9 +143,12 @@ def __init__(
                 sum_pips=sum_pips,
             )
             # Write result
-            result.df.write.mode(session.write_mode).parquet(
-                output_path + "/" + study_locus_to_finemap
-            )
+            if result is not None:
+                result.df.write.mode(session.write_mode).parquet(
+                    output_path + "/" + study_locus_to_finemap
+                )
+            else:
+                return
```

Comment:
Are you sure that it should be just empty return? Not "return None"? Or even without "else"?

Reply:
I think `return` and `return None` are equivalent. You're right that the else statement isn't required - I thought it might throw an error with mypy.